generators/generator-base.js

```diff
@@ -180,6 +180,32 @@ module.exports = class JHipsterBaseGenerator extends PrivateBase {
         return prettierExtensions;
     }
 
+    /**
+     * Replace placeholders with versions from packageJsonSourceFile.
+     * @param {string} keyToPlace - PlaceHolder name.
+     * @param {string} packageJsonSourceFile - Package json filepath with actual versions.
+     */
+    replacePackageJsonVersions(keyToReplace, packageJsonSourceFile) {
+        const packageJsonSource = JSON.parse(fs.readFileSync(packageJsonSourceFile, 'utf-8'));
+        const packageJsonFile = this.destinationPath('package.json');
+        const packageJson = this.fs.readJSON(packageJsonFile);
```

Comment:
That's a lot of packageJson variables :) Since we have a source, I guess it would be clearer to have target.
```suggestion
        const packageJsonTargetFile = this.destinationPath('package.json');
        const packageJsonTarget = this.fs.readJSON(packageJsonFile);
```